BSB_LAN/src/PubSubClient/library.json

```diff
@@ -1,18 +1,19 @@
 {
-    "name": "PubSubClient",
-    "keywords": "ethernet, mqtt, m2m, iot",
+    "name": "TBPubSubClient",
+    "keywords": "ethernet, mqtt, m2m, iot, thingsboard, messages",
     "description": "A client library for MQTT messaging. MQTT is a lightweight messaging protocol ideal for small devices. This library allows you to send and receive MQTT messages. It supports the latest MQTT 3.1.1 protocol and can be configured to use the older MQTT 3.1 if needed. It supports all Arduino Ethernet Client compatible hardware, including the Intel Galileo/Edison, ESP8266 and TI CC3000.",
     "repository": {
         "type": "git",
-        "url": "https://github.com/knolleary/pubsubclient.git"
+        "url": "https://github.com/thingsboard/pubsubclient.git"
     },
-    "version": "2.8",
+    "version": "2.10.0",
```

Comment:
for some reason they forgot to update the version number in the JSON file of release.

Reply:
they already forgot this on last release 2.10.1. I don't know for what the JSON file is used.